src/FSLibrary/MissionSimulatePubnet.fs

```diff
@@ -42,6 +42,116 @@ let simulatePubnet (context: MissionContext) =
                           }
                       )
                   )
+              loadGenOpCount = Some(context.loadGenOpCount |> Option.defaultValue (seq { 1 .. 100 }))
```

Comment:
Better to use the size of loadGenOpCountDistribution here

Reply:
As part of the update, I included the default `loadGenOpCount` in the file